.github/workflows/reusable-build-flavor.yaml

```diff
@@ -73,6 +73,7 @@ jobs:
           FLAVOR: ${{ inputs.flavor }}
           IMAGE: quay.io/kairos/core-${{ inputs.flavor }}:latest
         run: |
+          exit 1
```

Comment:
Notification only runs on master, so I guess we can drop this ... otherwise if you want to just test it, you can copy the notification into image-pr.yaml without the branch checks

Reply:
ffffuuuuuuuuuuuu

Reply:
fixed, good catch yikes